docs/docs/examples/workflow/router_query_engine.ipynb

```diff
@@ -0,0 +1,721 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #131.                return StreamingResponse(
</pre><p>Probably we would want to return AsyncStreamingResponse, since we used async <code>aget_response()</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/15635/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>